src/runtime/native/runtime.ml

```diff
@@ -10,16 +10,33 @@ let default_bisect_file = ref "bisect"
 
 let default_bisect_silent = ref "bisect.log"
 
+let sigterm_enable = ref false
+
+let sigterm_ignore_overriden_warned = ref false
+
+let sigterm_ignore_handle_warned = ref false
+
+let bisect_file_written = ref false
```

Comment:
We need to track whether the file has already been written. To terminate the application when receiving `SIGTERM`, the signal handler calls `exit`, triggering the `at_exit` hook. To avoid this, we set `bisect_file_written` to true in `dump`, and do nothing when it is set to true.